core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -313,12 +313,14 @@ object ConsumerGroupCommand {
 
     protected def describeGroup(group: String) {
       val consumerSummaries = adminClient.describeConsumerGroup(group)
-      if (consumerSummaries.isEmpty)
-        println(s"Consumer group `${group}` does not exist or is rebalancing.")
+      if (consumerSummaries == None)
+        println(s"Consumer Group `${group}` does not exist")
```

Comment:
Can you change "Group" to "group" in the print message to be consistent with previous format?